com.unity.netcode.gameobjects/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -128,10 +128,10 @@ public class NetworkConfig
         public int LoadSceneTimeOut = 120;
 
         /// <summary>
-        /// The amount of time a message should be buffered for without being consumed. If it is not consumed within this time, it will be dropped.
+        /// The amount of time a message should be buffered if the asset or object needed to process it doesn't exist yet. If the asset is not added/object is not spawned within this time, it will be dropped.
         /// </summary>
-        [Tooltip("The amount of time a message should be buffered for without being consumed. If it is not consumed within this time, it will be dropped")]
-        public float MessageBufferTimeout = 20f;
+        [Tooltip("The amount of time a message should be buffered if the asset or object needed to process it doesn't exist yet. If the asset is not added/object is not spawned within this time, it will be dropped")]
+        public float SpawnTimeout = 1f;
```

Comment:
perhaps?
```suggestion
        public float SpawnTimeoutSec = 1f;
```

Reply:
sure :(

Reply:
okay :(

Reply:
should we add changelog entries like "removed MessageBufferTimeout from NetworkConfig" and rephrase other added changelog entry to mention SpawnTimeout somehow there too?

Reply:
I don't think it's really worth mentioning the removal of MessageBufferTimeout given that the message buffers were removed before we ever released 1.0-pre, so that config was already dead and had no function. I removed it because it was dead code.

Reply:
I'll update the changelog to mention SpawnTimeout though.